bridges/ContainerLinuxReleasesBridge.php

```diff
@@ -40,18 +40,29 @@ public function collectData() {
 
 			$item['uri'] = "https://coreos.com/releases/#$releaseVersion";
 			$item['title'] = $releaseVersion;
-			$item['content'] = nl2br($release['release_notes']);
 
-			$item['content'] .= <<<EOT
-<br/>
+			$content = $release['release_notes'];
+			$content .= <<<EOT
+
 Major Software:
-<br/>
-- Kernel: {$release['major_software']['kernel'][0]}<br/>
-- Docker: {$release['major_software']['docker'][0]}<br/>
-- etcd: {$release['major_software']['etcd'][0]}<br/>
+* Kernel: <tt>{$release['major_software']['kernel'][0]}</tt>
+* Docker: <tt>{$release['major_software']['docker'][0]}</tt>
+* etcd: <tt>{$release['major_software']['etcd'][0]}</tt>
 EOT;
 			$item['timestamp'] = strtotime($release['release_date']);
 
+			// Based on https://gist.github.com/jbroadway/2836900
+			$regex = '/\[([^\[]+)\]\(([^\)]+)\)/';
+			$replacement = '<a href=\'\2\'>\1</a>';
+
+			$item['content'] = preg_replace($regex, $replacement, nl2br($content));
+
+			$regex = '/\n[\*|\-](.*)/';
```

Comment:
The regex can be improved slightly in order to correctly format all items in the feed:

![image](https://user-images.githubusercontent.com/5776685/41991513-ee2c3d26-7a45-11e8-9bd7-05449303743c.png)

Some elements do have spaces between newline and hyphen. You'll catch all of them with this regex:
`/\n\s*[\*|\-](.*)/`

Reply:
Had filed this in hurry, was wondering why this wasn't working.

Reply:
Let me know when you think this should be merged